src/console_cmds.cpp

```diff
@@ -2322,137 +2322,137 @@ DEF_CONSOLE_CMD(ConDumpInfo)
-	IConsoleCmdRegister("script",       ConScript);
-	IConsoleCmdRegister("scrollto",     ConScrollToTile);
-	IConsoleCmdRegister("alias",        ConAlias);
-	IConsoleCmdRegister("load",         ConLoad);
-	IConsoleCmdRegister("rm",           ConRemove);
-	IConsoleCmdRegister("save",         ConSave);
-	IConsoleCmdRegister("saveconfig",   ConSaveConfig);
-	IConsoleCmdRegister("ls",           ConListFiles);
-	IConsoleCmdRegister("cd",           ConChangeDirectory);
-	IConsoleCmdRegister("pwd",          ConPrintWorkingDirectory);
-	IConsoleCmdRegister("clear",        ConClearBuffer);
-	IConsoleCmdRegister("setting",      ConSetting);
-	IConsoleCmdRegister("setting_newgame", ConSettingNewgame);
-	IConsoleCmdRegister("list_settings",ConListSettings);
-	IConsoleCmdRegister("gamelog",      ConGamelogPrint);
-	IConsoleCmdRegister("rescan_newgrf", ConRescanNewGRF);
-
-	IConsoleAliasRegister("dir",          "ls");
-	IConsoleAliasRegister("del",          "rm %+");
-	IConsoleAliasRegister("newmap",       "newgame");
-	IConsoleAliasRegister("patch",        "setting %+");
-	IConsoleAliasRegister("set",          "setting %+");
-	IConsoleAliasRegister("set_newgame",  "setting_newgame %+");
-	IConsoleAliasRegister("list_patches", "list_settings %+");
-	IConsoleAliasRegister("developer",    "setting developer %+");
-
-	IConsoleCmdRegister("list_ai_libs", ConListAILibs);
-	IConsoleCmdRegister("list_ai",      ConListAI);
-	IConsoleCmdRegister("reload_ai",    ConReloadAI);
-	IConsoleCmdRegister("rescan_ai",    ConRescanAI);
-	IConsoleCmdRegister("start_ai",     ConStartAI);
-	IConsoleCmdRegister("stop_ai",      ConStopAI);
-
-	IConsoleCmdRegister("list_game",    ConListGame);
-	IConsoleCmdRegister("list_game_libs", ConListGameLibs);
-	IConsoleCmdRegister("rescan_game",    ConRescanGame);
-
-	IConsoleCmdRegister("companies",       ConCompanies);
-	IConsoleAliasRegister("players",       "companies");
+	IConsole::CmdRegister("debug_level",  ConDebugLevel);
+	IConsole::CmdRegister("echo",         ConEcho);
+	IConsole::CmdRegister("echoc",        ConEchoC);
+	IConsole::CmdRegister("exec",         ConExec);
+	IConsole::CmdRegister("exit",         ConExit);
+	IConsole::CmdRegister("part",         ConPart);
+	IConsole::CmdRegister("help",         ConHelp);
+	IConsole::CmdRegister("info_cmd",     ConInfoCmd);
+	IConsole::CmdRegister("list_cmds",    ConListCommands);
+	IConsole::CmdRegister("list_aliases", ConListAliases);
+	IConsole::CmdRegister("newgame",      ConNewGame);
+	IConsole::CmdRegister("restart",      ConRestart);
+	IConsole::CmdRegister("reload",       ConReload);
+	IConsole::CmdRegister("getseed",      ConGetSeed);
+	IConsole::CmdRegister("getdate",      ConGetDate);
+	IConsole::CmdRegister("getsysdate",   ConGetSysDate);
+	IConsole::CmdRegister("quit",         ConExit);
+	IConsole::CmdRegister("resetengines", ConResetEngines, ConHookNoNetwork);
+	IConsole::CmdRegister("reset_enginepool", ConResetEnginePool, ConHookNoNetwork);
```

Comment:
Might it be aesthetically worth the effort to line all the second parameters out/up under each other when the lines are getting changed anyway?

Reply:
I used global search and replace for them, but I can manually fix the alignment :)